cli/tests/test_git_push.rs

```diff
@@ -1440,36 +1440,57 @@ fn test_git_push_changes_with_name_untracked_or_forgotten(subprocess: bool) {
     // Make sure push still errors if we try to push a bookmark with the same name
     // to a different location.
     let output = work_dir.run_jj(["git", "push", "--named", "b1=@+"]);
-    insta::allow_duplicates! {
-    insta::assert_snapshot!(output, @r"
-    ------- stderr -------
-    Changes to push to origin:
-      Add bookmark b1 to c9c824c88955
-    Error: Failed to push some bookmarks
-    Hint: The following references unexpectedly moved on the remote:
-      refs/heads/b1 (reason: stale info)
-    Hint: Try fetching from the remote, then make the bookmark point to where you want it to be, and push again.
-    [EOF]
-    [exit status: 1]
-    ");
+    if subprocess {
+        insta::assert_snapshot!(output, @r"
+        ------- stderr -------
+        Changes to push to origin:
+          Add bookmark b1 to c9c824c88955
+        Error: Failed to push some bookmarks
+        Hint: The following references unexpectedly moved on the remote:
+          refs/heads/b1 (reason: stale info)
+        Hint: Try fetching from the remote, then make the bookmark point to where you want it to be, and push again.
+        [EOF]
+        [exit status: 1]
+        ");
+    } else {
+        insta::assert_snapshot!(output, @r"
+        ------- stderr -------
+        Changes to push to origin:
+          Add bookmark b1 to c9c824c88955
+        [EOF]
+        ");
     }
 
     // The bookmark is still forgotten
     let output = work_dir.run_jj(["bookmark", "list", "--all", "b1"]);
-    insta::allow_duplicates! {
-    insta::assert_snapshot!(output, @"");
+    if subprocess {
+        insta::assert_snapshot!(output, @"");
+    } else {
+        insta::assert_snapshot!(output, @r"
+        b1: yostqsxw c9c824c8 (empty) child
+          @origin: yostqsxw c9c824c8 (empty) child
+        [EOF]
+        ");
     }
     // In this case, pushing the bookmark to the same location where it already is
     // succeeds. TODO: This seems pretty safe, but perhaps it should still show
     // an error or some sort of warning?
     let output = work_dir.run_jj(["git", "push", "--named", "b1=@"]);
-    insta::allow_duplicates! {
-    insta::assert_snapshot!(output, @r"
-    ------- stderr -------
-    Changes to push to origin:
-      Add bookmark b1 to 10b6b209c4a3
-    [EOF]
-    ");
+    if subprocess {
```

Comment:
I don't think this diff is meaningful, as said in the above comment, but it's possible that fixing it will result in different, meaningful, diff similar to the first diff above.